Adversarial audio examples (#318)
IBM/adversarial-robustness-toolbox: notebooks/README.md

# Adversarial Robustness Toolbox notebooks

## Audio

[adversarial_audio_examples.ipynb](adversarial_audio_examples.ipynb) [[on nbviewer](https://nbviewer.jupyter.org/github/IBM/adversarial-robustness-toolbox/blob/master/notebooks/adversarial_audio_examples.ipynb)]

> show how to use ART to employ adversarial attacks on audio data.

Reviewer: ```suggestion
shows how to create adversarial examples of audio data with ART.
```
